### ISSUE TITLE
Must LoggerSeverity (namespace) needed to be exposed?

### ISSUE THREAD

**Issue by bitPogo** [asciidoctor/asciidoctor.js#1114]

**Comment by Mogztter** [comment]
Are you saying that `Asciidoctor.LoggerSeverity` is not available/visible in TypeScript?

https://github.com/asciidoctor/asciidoctor.js/blob/3384a6bdab7addc8095aa0ff6fabd3cb2e276946/packages/core/types/index.d.ts#L394-L396

**Comment by bitPogo** [comment]
Nope...it should be useable, but I see no place where it is actually used.

**Comment by Mogztter** [comment]
OK, I didn't know :smile: 

I guess we could add a test in `core/types/tests.ts`. This API can be useful when you want to use a custom logger. You can learn more about that here: https://github.com/asciidoctor/asciidoctor.js/blob/cb0db00597ae7ecfd65cc717f3e67f8c284d5abe/docs/modules/processor/pages/logging-api.adoc#custom-logger

**Comment by bitPogo** [comment]
It's actually usage is only internal, since the namespace is not ported to the outside...at least not in typescript. However the [code](https://github.com/asciidoctor/asciidoctor.js/blob/cb0db00597ae7ecfd65cc717f3e67f8c284d5abe/packages/core/src/asciidoctor-core-api.js#L2750-L2760) is speaking about it. The question is, if it should be exposed (so adding to the Asciidoctor class LoggerSeverity) or let it internal (so the type definition is dead code and should be delete, since it is basically just there).

**Comment by bitPogo** [comment]
Ah...so my first answer was wrong...`Asciidoctor.LoggerSeverity` is not present. Sorry a long day...I rewrote the whole types for kotlin...so thats why I stumbled over all those stuff...

**Comment by ggrossetie** [comment]
The code has been completely rewritten from the Ruby source, and the types are generated from JSDoc. LoggerSeverity is no longer exported (i.e., it is no longer part of the public API).

If you think something useful is missing in the next version of Asciidoctor.js (4.x), feel free to reopen an issue.
